sky/cli.py

```diff
@@ -1876,6 +1875,51 @@ def down(
                            purge=purge)
 
 
+def _hints_for_down_spot_controller(controller_name: str):
+    # spot_jobs will be empty when the spot cluster is not running.
+    cluster_status, _ = backend_utils.refresh_cluster_status_handle(
+        controller_name)
+    if cluster_status is None:
+        click.echo('Managed spot controller has already been torn down.')
+        return
+
+    msg = (f'{colorama.Fore.YELLOW}WARNING: Tearing down the managed '
+           f'spot controller ({cluster_status.value}). Please be '
+           f'aware of the following:{colorama.Style.RESET_ALL}'
+           '\n * All logs and status information of the spot '
+           'jobs will be lost.')
+    if cluster_status == global_user_state.ClusterStatus.UP:
+        try:
+            spot_jobs = core.spot_status(refresh=False)
+        except exceptions.ClusterNotUpError:
+            # The spot controller cluster status changed during querying
+            # the spot jobs, use the latest cluster status, so that the
+            # message for INIT and STOPPED states will be correctly
+            # added to the message.
+            cluster_status = backend_utils.refresh_cluster_status_handle(
+                controller_name)
+            spot_jobs = []
+
+        # Find in-progress spot jobs, and hint users to cancel them.
+        non_terminal_jobs = [
+            job for job in spot_jobs if not job['status'].is_terminal()
+        ]
+        if (cluster_status == global_user_state.ClusterStatus.UP and
+                non_terminal_jobs):
+            msg += ('\n * In-progress spot jobs found, their resources '
+                    'will not be terminated and require manual cleanup '
+                    '(sky spot cancel --all):\n')
```

Comment:
Do we want to show (sky spot cancel --all)? User is downing the controller, and after it's downed, spot cancel won't work? I feel either of the following is fine:
- just say manual cleanup (I prefer this one)
- say manual cleanup via cloud consoles

Reply:
Ahh, I was trying to say abort the current down process and run `sky spot cancel --all` first, and then do the `sky down`, but that could take a long time, waiting the controller process hit the line that checks the cancel signal.

Removed. Thanks!